src/System.Private.CoreLib/src/System/Diagnostics/Eventing/RuntimeEventSourceHelper.Linux.cs

```diff
@@ -0,0 +1,72 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Buffers;
+using System.IO;
+
+
+namespace System.Diagnostics.Tracing
+{
+	internal sealed class RuntimeEventSourceHelper
+	{
+		internal static long GetProcessTimes()
+		{
+            long pid = Environment.GetCurrentProcessId();
+            using (FileStream fs = new FileStream($"/proc/{pid}/stat", FileMode.Open, FileAccess.Read, FileShare.Read, bufferSize: 1))
+            {
+                byte[] text = ReadAllBytesUnknownLength(fs);
+                string s = System.Text.Encoding.UTF8.GetString(text, 0, text.Length);
+                long _user = Convert.ToInt64(s.Split(" ")[13]);
+                long _system = Convert.ToInt64(s.Split(" ")[14]);
```

Comment:
Split is relatively expensive.

Reply:
@stephentoub What would you recommend? What would be a performant option here? Using a regex?

Reply:
None of this should be needed once https://github.com/dotnet/coreclr/pull/22901#discussion_r261051884 is addressed.